frontend/pages/policies/details/components/PolicyAutomationActivityDetailsModal/_styles.scss

```
@@ -0,0 +1,44 @@
+  }
+
+  &__row {
+    display: flex;
+    gap: $pad-xxlarge;
+  }
+
+  &__details-label {
+    font-weight: $bold;
+    color: $core-fleet-black;
+
+    display: flex;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)